include/kernel.h

```diff
@@ -1934,10 +1919,11 @@ static inline int z_impl_k_queue_is_empty(struct k_queue *queue)
  */
 __syscall void *k_queue_peek_head(struct k_queue *queue);
 
-static inline void *z_impl_k_queue_peek_head(struct k_queue *queue)
+/*static inline void *z_impl_k_queue_peek_head(struct k_queue *queue)
```

Comment:
This code commented out here, just to be removed later in this commit: https://github.com/zephyrproject-rtos/zephyr/pull/34843/commits/e9c9e73e5d62134037831dce3c960326d516e3b9

Why not remove it directly in https://github.com/zephyrproject-rtos/zephyr/pull/34843/commits/ef9c5f0f5b1b1bfe959b00c4d376fd915a17c898.